Lean4Axiomatic/Natural/Exponentiation.lean

```diff
@@ -95,7 +97,7 @@ theorem pow_substL {x₁ x₂ : α} {m : ℕ} : x₁ ≃ x₂ → x₁ ^ m ≃ x
     show x₁ ^ 0 ≃ x₂ ^ 0
     calc
       _ ≃ x₁ ^ 0 := Rel.refl
-      _ ≃ 1      := pow_zero
+      _ ≃ ident  := pow_zero
```

Comment:
If you create an `OfNat α 1` instance above `Exponentiation.Props` (in the same place that I recommended putting the `Mul` instance), setting `ofNat := ident`, then you won't need to replace `1` with `ident` here or in any of the rest of the file.